release.sh

```diff
@@ -1,23 +1,27 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-set -v -e -o pipefail
+set -eux -o pipefail
 
 VERSION=$(git describe --abbrev=0 --tags)
 ARCH="${ARCH:-64}"
-CUSTOM_FLAGS=""
-
+CUSTOM_FLAGS=()
 unameOut="$(uname -s)"
 case "$unameOut" in
     Linux*)
         OS=linux
-        CUSTOM_FLAGS="-L--export-dynamic"
+        CUSTOM_FLAGS+=("-L--export-dynamic")
         ;;
     Darwin*)
         OS=osx
+        CUSTOM_FLAGS+=("-L-macosx_version_min" "-L10.7" "-L-lcrt1.o")
```

Comment:
From: https://github.com/dlang-community/discussions/issues/9#issuecomment-378725199

Reply:
Maybe even 10.9? (as this will come with 2.079)

https://dlang.org/changelog/2.079.0.html#osx-10_9

Reply:
BTW what for do we need `lcrt1.o`?
I assumed this is a legacy thing too?